charts/docker-hub-rate-limit-exporter/values.yaml

```diff
@@ -8,7 +8,7 @@ config:
     dockerhubUsername: ## put your dockerhub username here
     dockerhubPassword: ## put your dockerhub password here
 
-replicaCount: 1
+replicaCount: 2
```

Comment:
What is the reason behind changing to 2 replicas? 
Is it just for high availability? I haven't tested the deployment with two replicas, so I think I would prefer to leave the default value as one replica? What do you think?

Reply:
I was just doing a demo. I'll put it back to 1.